protocol-units/da-sequencer/node/src/batch.rs

```diff
@@ -1,182 +1,136 @@
 use crate::error::DaSequencerError;
+use crate::whitelist::Whitelist;
+use movement_da_sequencer_config::DaSequencerConfig;
 use aptos_crypto_derive::{BCSCryptoHash, CryptoHasher};
 use bcs;
-use ed25519_dalek::{Signature, Verifier, VerifyingKey};
+use ed25519_dalek::{Signature, Signer, Verifier, VerifyingKey};
 use movement_types::transaction::Transaction;
 use serde::{Deserialize, Serialize};
 use std::ops::Deref;
 
 #[derive(Debug, Clone)]
 pub struct RawData {
-	pub data: Vec<u8>,
+        pub data: Vec<u8>,
```

Comment:
Please don't reformat files like that. This brings a lot of unneeded diff noise. Check that your editor can pick up settings from `rustfmt.toml`.

NB: it would be good, in my opinion, to switch to space indentation, but use the standard 4 spaces.